client_sdk/go/samples/admin/main.go

```diff
@@ -0,0 +1,347 @@
+/*
```

Comment:
I guess this code is eventually going to replace the `setup` code from below (`utils.go` if i remember correctly)?  If so i would keep the testing code in integration tests and not trying to add samples already in this PR.

This also would remove the need for the `testing` package which too me doesn't seem that motivated as a package. Also if this really is a sample code, then it shouldn't rely on internal packages and anything samples?  If it is really testing (and as mentioned above it should hence be moved to `integration/....` then there is no need for a global testing package ...

Reply:
I removed the dependency to `internal/testing`